shell/common/node_bindings.cc

```diff
@@ -188,16 +189,26 @@ void ErrorMessageListener(v8::Local<v8::Message> message,
   }
 }
 
+const std::unordered_set<base::StringPiece, base::StringPieceHash>
+GetAllowedDebugOptions() {
```

Comment:
Rename to GetAllowedCliInspect for consistency with the fuse name (or rename the fuse to IsNodeCliDebugEnabled?)

Reply:
`DebugOptions` was for consistency with what keys they map to inside of `NodeOptions` --> `DebugOptions` is an actual internal struct.  Of which the "inspect" CLI flags are some of them.

The method is for getting the allowed node debug options, of which some _may_ be inspect options.  I think the relation between the fuse and method here could be clearer but the naming is accurate.